recipe/conda_build_config.yaml

```diff
@@ -252,7 +252,7 @@ pin_run_as_build:
     max_pin: x.x
   libuuid:
     max_pin: x
-  lz4-c:
+  lz4_c:
```

Comment:
I think the dash here is needed, ironically. :/

Reply:
ok.